ShareTheTable/View Controllers/ResultsViewController.m

```diff
@@ -0,0 +1,120 @@
+//
+//  ResultsViewController.m
+//  ShareTheTable
+//
+//  Created by Andrea Gonzalez on 7/12/22.
+//
+
+#import "ResultsViewController.h"
+#import "Parse/Parse.h"
+#import "UserCell.h"
+#import "User.h"
+#import "NSData+Base64.h"
+#import "SearchViewController.h"
+
+@interface ResultsViewController () <UITableViewDelegate, UITableViewDataSource>
+
+@end
+
+@implementation ResultsViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    
+    self.userTableView.dataSource = self;
+    self.userTableView.delegate = self;
+    self.userTableView.rowHeight = UITableViewAutomaticDimension;
+    
+    UIRefreshControl *refreshControl = [[UIRefreshControl alloc] init];
+    [refreshControl addTarget:self action:@selector(beginRefresh:) forControlEvents:UIControlEventValueChanged];
+    [self.userTableView insertSubview:refreshControl atIndex:0];
+}
+
+- (void)beginRefresh:(UIRefreshControl *)refreshControl {
+    [self refreshData];
+
+    // Reload the tableView now that there is new data
+    [self.userTableView reloadData];
+
+    // Tell the refreshControl to stop spinning
+    [refreshControl endRefreshing];
+}
+
+- (void)didPush:(nonnull User *)user {
+    //[self.arrayOfPosts insertObject:post atIndex:0];
+    [self.userTableView reloadData];
+}
+
+- (void) loadMoreData {
```

Comment:
"loadMoreData" isn't a very descriptive method name - think of someone looking at your code for the first time, you want them to be able to follow very easily. Better method names + a little comment about what the method does will help.